src/olympia/addons/serializers.py

```diff
@@ -37,6 +37,46 @@ def get_url(self, obj):
         return obj.get_url_path(src='')
 
 
+class PreviewSerializer(serializers.ModelSerializer):
+    caption = TranslationSerializerField()
+    image_url = serializers.SerializerMethodField()
+    thumbnail_url = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Preview
+        fields = ('id', 'caption', 'image_url', 'thumbnail_url')
+
+    def get_image_url(self, obj):
+        return absolutify(obj.image_url)
+
+    def get_thumbnail_url(self, obj):
+        return absolutify(obj.thumbnail_url)
+
+    # FIXME:
```

Comment:
I'm not a fan of comments like that. Can we at least move that comment into the class docstring and create an appropriate issue for it and link it in the code?